packages/frontend/src/components/Card/Card.styled.ts

```diff
@@ -0,0 +1,115 @@
+import styled, { css } from "styled-components";
+
+import { Color } from "src/enums/color.enum";
+import { FontSize } from "src/enums/fontsize.enum";
+import { FontWeight } from "src/enums/fontweight.enum";
+import svg from "react-inlinesvg";
+import { rgba } from "polished";
+import { Link } from "react-router-dom";
+import { zIndex } from "src/enums/zIndex.enum";
+import { Breakpoint } from "src/enums/breakpoint.enum";
+import { HeartIconTypes, ImageContainerTypes } from "./Card.types";
+
+export const ImageContainer = styled.div<ImageContainerTypes>`
```

Comment:
już widziałem taki kod z przyciemnianiem wrzuć taki komponent do common/theme i rozszerzaj

Reply:
sam ImageContainer nie, tylko ten ::before został użyty raz z innym border radiusem i background colorem, wiec nie wiem czy robić to jakoś reużywalne?

Reply:
niech zotanie narzie jka jest lae jak pojawi się jescze raz to ja by wydzielił